api/nakadi-event-bus-api.yaml

```diff
@@ -416,28 +416,35 @@ paths:
       security:
         - oauth2: ['nakadi_event_stream.write']
       description: |
-        Publishes one `Event` of this `EventType`. (**Note** *Format and response semantics of publishing of
-        batches of Events will be added in the short term.*)
+        Publishes a batch `Event`s of this `EventType`. All items must be of the same EventType.
```

Comment:
I think it is not only the *same* envettype but also the event type specified by the `{name}`